components/footer.js

```diff
@@ -1 +1,20 @@
-// footer component
+import Link from 'next/link';
+import styles from './footer.module.css';
+import utilsStyles from '../styles/utils.module.css';
+
+export default function Footer() {
+	return (
+		<footer>
+			<p className={`${utilsStyles.font_white} ${styles.text}`}>
+				Copyright ©2021.{' '}
+				<div className={styles.link_wrap}>
+					<Link href="https://github.com/chingu-voyages/v35-geckos-team-08">
```

Comment:
Thank you for putting link tag to this GitHub page!

Reply:
Yup! 

@Shiho317 @bluwakpa 
Actually, I noticed that you are not supposed to nest `<div>` inside `<p>` after I created this PR. (There should be an error message on the console.)
I fixed it and you should see the correct version of this code on the next PR.
Just so you know!

Reply:
Thanks for giving our team credit! Idk about the nesting rule either. Makes sense!